ddclient

```diff
@@ -2951,11 +2951,11 @@ o 'concont'
 
 The 'concont' protocol is the protocol used by the content management
 system ConCont's dydns module. This is currently used by the free
```

Comment:
Does this need to change, too?

Reply:
It's unclear. If you look at the URL you'll see `dydns` in there, too. That makes me think there used to be a software product called ConCont that did actually have something spelled `dydns`. I can't find anything about ConCont, however.

Perhaps it's time to delete the `concont` protocol?

Reply:
Maybe the comment needs an update. I am not sure what we should do here.